scene/gui/spin_box.cpp

```diff
@@ -39,7 +39,7 @@ Size2 SpinBox::get_minimum_size() const {
 	return ms;
 }
 
-void SpinBox::_value_changed(double) {
+void SpinBox::_value_changed(double p_value) {
 	String value = TS->format_number(String::num(get_value(), Math::range_step_decimals(get_step())));
```

Comment:
Could be material for another PR, but seems weird to rely on `get_value()` here instead of actually using the value passed to the method.